programs/clearing_house/src/state/user.rs

```diff
@@ -159,6 +159,15 @@ pub struct MarketPosition {
     pub open_bids: i128,
     pub open_asks: i128,
 
+    // lp stuff
+    pub lp_shares: u128,
+    pub lp_base_asset_amount: i128,
+    pub lp_quote_asset_amount: u128,
+    pub last_cumulative_funding_payment_per_lp: i128,
+    pub last_cumulative_fee_per_lp: u128,
+    pub last_cumulative_net_base_asset_amount_per_lp: i128,
+    pub last_lp_add_time: i64,
```

Comment:
maybe we should generalize this to be a general `ts` for position which tracks the last time you interacted with a position (not critical)

Reply:
yea would be nice to enforce a cool down period for pnl settling as well (could be very short but not faster than a liq could get discovered)